contracts/staking/libs/Rebates.sol

```diff
@@ -88,15 +103,29 @@ library Rebates {
 
         // Calculate the rebate rewards for the indexer
         if (pool.fees > 0 && pool.effectiveAllocatedStake > 0) {
-            rebateReward = LibCobbDouglas.cobbDouglas(
-                pool.fees, // totalRewards
-                _indexerFees,
-                pool.fees,
-                _indexerEffectiveAllocatedStake,
-                pool.effectiveAllocatedStake,
-                pool.alphaNumerator,
-                pool.alphaDenominator
-            );
+            if (pool.lambdaDenominator > 0) {
+                // Exponential rebates
+                rebateReward = LibExponential.exponentialRebates(
+                    _indexerFees,
+                    _indexerEffectiveAllocatedStake,
+                    pool.alphaNumerator,
+                    pool.alphaDenominator,
+                    pool.lambdaNumerator,
+                    pool.lambdaDenominator
+                );
+            } else {
+                // (Deprecated) Cobb-Douglas rebates
+                // Keeping this logic to allow existing open pools to be closed
```

Comment:
Maybe we can find a different upgrade path so that we don't need this code. What if use the new formula for old rebates?

Reply:
I thought about that, but is it _fair_ to indexers? Going to ask Justin/Howard about this.